src/sql/azdata.proposed.d.ts

```diff
@@ -597,6 +597,11 @@ declare module 'azdata' {
 		 * Authentication token for the current session.
 		 */
 		securityToken?: accounts.AccountSecurityToken | undefined;
+
+		/**
+		 * Filter for expanding node.
+		 */
+		filters?: NodeFilter[];
```

Comment:
It is very likely that users will request for AND/OR and condition grouping support, until we have that, I think we should support the `Matches` operator so that users can use Regex for complex conditions meanwhile it is easy for us to implement.

Reply:
Currently, smo urn filters do not have regex support to query objects. For now, it will be out of scope of may release